gyb_syntax_support/Child.py

```diff
@@ -130,6 +130,9 @@ def generate_default_initialization(self):
       if self.token.text:
         return f" = .{self.token.swift_kind()}Token()"
       if self.text_choices and len(self.text_choices) == 1:
-        return f" = .{self.token.swift_kind()}(\"{self.text_choices[0]}\")"
+        if self.token.associated_value_class:
+          return f" = .{self.token.swift_kind()}(.{self.text_choices[0]})"
+        else:
+          return f" = .{self.token.swift_kind()}(\"{self.text_choices[0]}\")"
```

Comment:
And this, is it also needed in the Child.swift

Reply:
Same, will be added by https://github.com/apple/swift-syntax/pull/1174